### PR TITLE
Edit the readme.md file v1

### PR DESCRIPTION
**Pull request by devrulls**
Edit the readme.md file v1

1. Prerequisites
2. Deployment
3. Authors
4. Technologies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,54 @@
...
+This application is deployed on heroku : https://id.heroku.com/login
 
-### *[alternative 2]*
+        link webapp:
 
-* `+` *[argument 1 pro]*
-* `+` *[argument 2 pro]*
-* `-` *[argument 1 con]*
-* *[...]* <!-- numbers of pros and cons can vary -->
+## Authors
```
Comment (Saxamos) [inline README.md]: Not necessary since contributors are written in the project :)

File: README.md [hunk 2]
```diff
@@ -1,76 +1,54 @@
-# kibblepi
+# Kibblepi - Developed at SolipsIA
 
-This project is ...
-Internship project ...
+This is an internship project developed at SolipsIA to validate the competences acquired in the professional title "Concepteur Développeur d’applications" at the GRETA Littoral - Montpellier.
+This project consists in the creation of a cat kibble distributor, called "Kibblepi", this distributor will allow you to feed your pet whenever you want, 
+through a web application developed with the Streamlit framework and server side developed with the micro-framework flask. 
+(Flask is a micro web framework written in Python. It is classified as a microframework because it does not require particular tools or libraries)
```
Comment (Saxamos) [inline README.md:6]: Nice presentation

File: README.md [hunk 3]
```diff
@@ -1,76 +1,54 @@
...
+- Accessory kit-Arduino
 
-## Decision Outcome
-
-* Chosen Alternative: *[alternative 1]*
-* *[justification. e.g., only alternative, which meets KO criterion decision driver | which resolves force force | ... | comes out best (see below)]*
-* *[consequences. e.g., negative impact on quality attribute, follow-up decisions required, ...]* <!-- optional -->
-
-## Pros and Cons of the Alternatives <!-- optional -->
+### Setup env
```
Comment (Saxamos) [inline README.md:40]: This one should be filled to be able to install deps and launch both streamlit and flask locally for any newjoiner :)

File: README.md [hunk 4]
```diff
@@ -1,76 +1,54 @@
...
 
-### *[alternative 1]*
+## Deployment
 
-* `+` *[argument 1 pro]*
-* `+` *[argument 2 pro]*
-* `-` *[argument 1 con]*
-* *[...]* <!-- numbers of pros and cons can vary -->
+This application is deployed on heroku : https://id.heroku.com/login
```
Comment (Saxamos) [inline README.md]: Maybe some explanation about the fact that it's deployed when push on main

File: README.md [hunk 5]
```diff
@@ -1,76 +1,54 @@
...
 
-### *[alternative 2]*
+        link webapp:
```
Comment (Saxamos) [inline README.md]: ?

**Review by Saxamos — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (devrulls) [reply README.md]: Modification made

File: README.md (hunk 5 above)
Comment (devrulls) [reply README.md]: https://kibblepi.herokuapp.com/

File: README.md (hunk 3 above)
Comment (devrulls) [reply README.md:40]: pipenv - modification made!

**Review by Saxamos — APPROVED** [review]

**Pull request merged**